packages/store/addon/-private/system/model/internal-model.ts

```diff
@@ -895,13 +902,20 @@ export default class InternalModel {
     @param {String} name
     @param {Object} context
   */
-  send(name, context?) {
+  send(name, context?, fromModel?) {
     let currentState = this.currentState;
 
     if (!currentState[name]) {
       this._unhandledEvent(currentState, name, context);
     }
 
+    if (true) {
+      if (fromModel && name === 'becameInvalid' && this._recordData.getErrors().length === 0) {
```

Comment:
if this lands before the Evented deprecation (seems likely even though that PR is 99% ready) then we'll need to address this there

cc #6059

Reply:
Not quite sure what you mean?

Reply:
just leaving a note for that PR because it is currently code complete just needs a name cleanup on the deprecations it introduces